dotnet/src/Connectors/Connectors.Onnx/Connectors.Onnx.csproj

```
@@ -4,7 +4,7 @@
     <!-- THIS PROPERTY GROUP MUST COME FIRST -->
     <AssemblyName>Microsoft.SemanticKernel.Connectors.Onnx</AssemblyName>
     <RootNamespace>$(AssemblyName)</RootNamespace>
-    <TargetFrameworks>net8.0;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks>net8.0;netstandard2.1</TargetFrameworks>
```

Suggest keeping the 2.0 or adding support for both 2.0 and 2.1.

```suggestion
    <TargetFrameworks>net8.0;netstandard2.0</TargetFrameworks>
```

---

`Microsoft.ML.OnnxRuntimeGenAI` only support 2.1

---

Just opened a PR bringing standard 2.0 to Onnx, hope they accept it.

Link: https://github.com/microsoft/onnxruntime-genai/pull/582

---

Unresolving the conversation because OnnxRuntimeGenAI was recently updated to support netstandard2.0.

---

There's no updated nupkg yet.

---

0.3.0 released with `netstandard2.0` support